org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/symbol/DocumentSymbolService.xtend

```diff
@@ -205,17 +206,17 @@ class DocumentSymbolService {
 		return description.qualifiedName.symbolName
 	}
 
-	protected def int getSymbolKind(IEObjectDescription description) {
+	protected def SymbolKind getSymbolKind(IEObjectDescription description) {
 		return description.EClass.symbolKind
 	}
 
 	protected def String getSymbolName(QualifiedName qualifiedName) {
 		return qualifiedName?.toString
 	}
 
-	protected def int getSymbolKind(EClass type) {
-		return 0
-//		return SymbolInformation.KIND_PROPERTY
```

Comment:
I guess because i like an icon for an undefined kind in VSCode more than an icon for a property as a default.